lib/datapacksexpand.js

```diff
@@ -425,6 +425,15 @@ DataPacksExpand.prototype.getNestedDataPackReferences = function(currentData, pa
                 }
                 
             } else {
+                if(!(jobInfo.maxDepth === 0) || jobInfo.includeSalesforceMetadata){
```

Comment:
This should probably be an && where `(jobInfo.maxDepth != 0 && jobInfo.includeSalesforceMetadata)` since if either false then it should not run.